cloud/kubernetes/rbac.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cockroachdb
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
GKE unfortunately requires an extra step before creating this `Role` and the `ClusterRole` is permitted (https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#prerequisites_for_using_role-based_access_control). I can update the README/docs for this if you don't want to.